Fix electron-builder 25 -> 26.
Add dependencies on grist-core and ext so npm list report correct graph.
Drop obsolete includeSubNodeModules.
Drop flatpak configuration which is now invalid (and not needed).
Exclude node_modules/grist-core to avoid duplicates.

Index: package.json
--- package.json.orig
+++ package.json
@@ -25,6 +25,10 @@
       "**/typescript-eslint/**"
     ]
   },
+  "dependencies": {
+    "grist-core": "*",
+    "ext": "*"
+  },
   "devDependencies": {
     "@types/trusted-types": "2.0.7",
     "@unrs/resolver-binding-wasm32-wasi": "^1.12.2",
@@ -59,7 +63,6 @@
     "appId": "com.getgrist.grist",
     "npmRebuild": false,
     "compression": "normal",
-    "includeSubNodeModules": true,
     "icon": "core/static/icons/grist.png",
     "win": {
       "artifactName": "${name}-${version}-${os}-${arch}.${ext}",
@@ -98,78 +101,6 @@
         "flatpak"
       ]
     },
-    "flatpak": {
-      "branch": "stable",
-      "runtime": "org.freedesktop.Platform",
-      "runtimeVersion": "25.08",
-      "sdk": "org.freedesktop.Sdk",
-      "base": "org.electronjs.Electron2.BaseApp",
-      "baseVersion": "25.08",
-      "finishArgs": [
-        "--share=ipc",
-        "--socket=x11",
-        "--socket=wayland",
-        "--socket=pulseaudio",
-        "--device=dri",
-        "--talk-name=org.freedesktop.Notifications",
-        "--talk-name=org.freedesktop.secrets",
-        "--filesystem=home",
-        "--allow=multiarch",
-        "--filesystem=xdg-cache/tmp:create",
-        "--env=TMPDIR=/tmp",
-        "--env=GRIST_SANDBOX_FLAVOR=pyodide"
-      ],
-      "desktop": {
-        "Name": "Grist",
-        "Comment": "Spreadsheet software to end data chaos",
-        "Icon": "com.getgrist.grist",
-        "Categories": [
-          "Office"
-        ],
-        "MimeType": "text/csv;text/tab-separated-values;text/plain;application/vnd.openxmlformats-officedocument.spreadsheetml.sheet;application/vnd.ms-excel.sheet.macroEnabled.12;"
-      },
-      "mimeTypes": [
-        "text/csv",
-        "text/tab-separated-values",
-        "text/plain",
-        "application/vnd.openxmlformats-officedocument.spreadsheetml.sheet",
-        "application/vnd.ms-excel.sheet.macroEnabled.12"
-      ],
-      "files": [
-        [
-          "dist/icons/16x16.png",
-          "/app/share/icons/hicolor/16x16/apps/com.getgrist.grist.png"
-        ],
-        [
-          "dist/icons/32x32.png",
-          "/app/share/icons/hicolor/32x32/apps/com.getgrist.grist.png"
-        ],
-        [
-          "dist/icons/48x48.png",
-          "/app/share/icons/hicolor/48x48/apps/com.getgrist.grist.png"
-        ],
-        [
-          "dist/icons/64x64.png",
-          "/app/share/icons/hicolor/64x64/apps/com.getgrist.grist.png"
-        ],
-        [
-          "dist/icons/128x128.png",
-          "/app/share/icons/hicolor/128x128/apps/com.getgrist.grist.png"
-        ],
-        [
-          "dist/icons/256x256.png",
-          "/app/share/icons/hicolor/256x256/apps/com.getgrist.grist.png"
-        ],
-        [
-          "dist/icons/512x512.png",
-          "/app/share/icons/hicolor/512x512/apps/com.getgrist.grist.png"
-        ],
-        [
-          "dist/icons/scalable.svg",
-          "/app/share/icons/hicolor/scalable/apps/com.getgrist.grist.svg"
-        ]
-      ]
-    },
     "afterSign": "./scripts/notarize.js",
     "asarUnpack": [
       "core/plugins",
@@ -207,6 +138,7 @@
       "!**/{appveyor.yml,.travis.yml,circle.yml}",
       "!**/{npm-debug.log,yarn.lock,.yarn-integrity,.yarn-metadata.json}",
       "!node_modules/ext",
+      "!node_modules/grist-core",
       "!**/node_modules/electron",
       "!**/node_modules/app-builder-bin",
       "!**/node_modules/7zip-bin",
